js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetGroupNode.tsx

```diff
@@ -75,8 +75,8 @@ export const AssetGroupNode = ({group, scale}: Props) => {
           position: 'absolute',
           background:
             scale < GROUPS_ONLY_SCALE
-              ? colorLineageGroupNodeBorder()
-              : colorLineageGroupNodeBorder(),
+              ? colorLineageGroupBackground()
+              : colorLineageGroupBackground(),
```

Comment:
Since both branches of the ternary are the same, you can remove the conditional:

```
background: colorLineageGroupBackground(),
```